src/components/RegisterAccount1.tsx

```diff
@@ -96,12 +97,12 @@ export const RegisterAccount1 = (props: RegisterAccount1Props) => {
           </Col>
           <Col xs={12} sm={8} className="blue-background">
             <img
-              className="sageLogo"
-              src={SageBionetworksLogo}
-              alt="Sage Bionetworks Logo"
-              style={{width: 270}}
-            />
-            <div className="registrationPanel">
+                className="sageLogo"
+                src={SageBionetworksLogo}
+                alt="Sage Bionetworks Logo"
+                style={{width: 270}}
+              />
+            <div className="mainContent">
```

Comment:
renamed registrationPanel to mainContent and moved duplicate styles to _SourceApp.scss